packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js

```diff
@@ -1,219 +1,289 @@
-      'entries',
-      '--org',
-      process.env.ORG,
-      '--stack-api-key',
-      process.env.STACK,
-      '--branch',
-      process.env.INVALID_BRANCH,
-    ];
-    await ExportToCsvCommand.run(args);
-    assert.calledWith(errorStub, 'Failed to fetch Branch. Please try again with valid parameters.');
-  });
-
-  it('Should throw an error when invalid contenttype is passed', async () => {
-    const args = [
-      '--action',
-      'entries',
-      '--org',
-      process.env.ORG,
-      '--stack-api-key',
-      process.env.STACK,
-      '--branch',
-      process.env.BRANCH,
-      '--content-type',
-      'invalid',
-      '--locale',
-      'en-us',
-    ];
-    await ExportToCsvCommand.run(args);
-    assert.calledWith(
-      errorStub,
-      `The Content Type invalid was not found. Please try again. Content Type is not valid.`,
-    );
-  });
-
-  it('Should throw an error when invalid locale is passed', async () => {
-    const args = [
-      '--action',
-      'entries',
-      '--org',
-      process.env.ORG,
-      '--stack-api-key',
-      process.env.STACK,
-      '--branch',
-      process.env.BRANCH,
-      '--content-type',
-      'header',
-      '--locale',
-      'invalid',
-    ];
-    await ExportToCsvCommand.run(args);
-    assert.calledWith(errorStub, 'Language was not found. Please try again.');
+  describe('Create taxonomies & terms csv file with prompt', () => {
+    test
+      .stdout({ print: process.env.PRINT === 'true' || false })
+      .stub(fs, 'createWriteStream', () => new PassThrough())
+      .stub(inquirer, 'registerPrompt', () => {})
+      .stub(inquirer, 'prompt', () => {
+        return Promise.resolve({
+          action: 'taxonomies',
+          chosenOrg: mockData.organizations[0].name,
+          chosenStack: mockData.stacks[0].name,
+        });
+      })
+      .nock(cma, (api) => {
+        api
+          .get(`/v3/stacks?&query={"org_uid":"${mockData.organizations[0].uid}"}`)
+          .reply(200, { stacks: mockData.stacks });
+      })
+      .nock(cma, (api) => {
+        api.get(`/v3/organizations?limit=100`).reply(200, { organizations: mockData.organizations });
+      })
+      .nock(cma, (api) => {
+        api
+          .get('/v3/taxonomies/taxonomy_uid_1?include_count=true&skip=0&limit=30')
+          .reply(200, { taxonomy: mockData.taxonomiesResp.taxonomies[0] });
+      })
+      .nock(cma, (api) => {
+        api
+          .get('/v3/taxonomies/taxonomy_uid_1/terms?include_count=true&skip=0&limit=100')
+          .reply(200, { terms: mockData.termsResp.terms, count: mockData.termsResp.count });
+      })
+      .command(['cm:export-to-csv', '--taxonomy-uid', 'taxonomy_uid_1'])
+      .it('CSV file should be created');
   });
 });
 
-describe('Export to csv command with action = users', () => {
-  let inquireStub;
-  let errorStub;
-  let consoleLogStub;
+describe('export-to-csv with action entries', () => {
+  if (!fs.existsSync(directory)) fs.mkdirSync(directory);
```

Comment:
Why are we creating a folder in the unit test case.?

Reply:
@antonyagustine removed